src/pages/Auth/ForgotPasswordPage.jsx

```diff
@@ -1,7 +1,61 @@
+import {Link} from 'react-router-dom';
+
+import FormContainer from '../../components/FormContainer';
+
 function ForgotPasswordPage() {
     return (
         <>
-            <h4 className="w-full text-center">Forgot Password Page</h4>
+             {/* Construction Image */}
+             <div className="w-full lg:w-3/6 px-4 hidden md:block">
+                <img src={require("../../assets/images/construction.svg").default} alt="Construction"></img>
+            </div>
+
+            {/* Login Form */}
```

Comment:
Correct the comment

Reply:
Ok

Reply:
Change the comment to Forgot Password Form

Reply:
Resolved

Reply:
I have commented on line 13. Meaning - Something has to be changed on line 13. 
'Login Form' has to be changed to 'Forgot Password Form'.

Reply:
Resolved